nucliadb_node/src/shards/metadata.rs

```diff
@@ -55,6 +56,7 @@ impl From<Similarity> for VectorSimilarity {
 pub struct ShardMetadata {
     kbid: Option<String>,
     similarity: Option<Similarity>,
+    pub channel: Channel,
 }
```

Comment:
there are already created shards  that do not have `channel` on their metadata so there are two choices here:
```rust
pub struct ShardMetadata {
    kbid: Option<String>,
    similarity: Option<Similarity>,
     #[serde(default)] 
    pub channel: Channel, // now if the value is not present it will use the default
}
```
or
```rust
pub struct ShardMetadata {
    kbid: Option<String>,
    similarity: Option<Similarity>,
    channel: Option<Channel>, // now if the value is not present it will use None
}
// In this approach a getter should be define 
```
The approach I like most is the first one, in fact maybe `similarity` should do the same